hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ContainerEndpoint.java

```diff
@@ -157,11 +157,17 @@ public Response getKeysForContainer(
       for (ContainerKeyPrefix containerKeyPrefix : containerKeyPrefixMap
           .keySet()) {
 
-        // Directly calling get() on the Key table instead of iterating since
-        // only full keys are supported now. When we change to using a prefix
-        // of the key, this needs to change to prefix seek.
-        OmKeyInfo omKeyInfo = omMetadataManager.getKeyTable(getBucketLayout())
-            .getSkipCache(containerKeyPrefix.getKeyPrefix());
+      // Directly calling get() on the Key table instead of iterating since
```

Comment:
Actually, we are calling 'getSkipCache()' not 'get()' and we aren't calling the function directly on the key table rather we are deciding the table i.e, file table or key table based on the bucket layout, maybe you can update the comment accordingly.

Reply:
Thanks! I have made the change.